modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationForm.testcase

```diff
@@ -0,0 +1,160 @@
+				userLoginFullName = "Employee Employee");
+		}
+
+		task ("Navigate to EVP Site") {
+			EVPUtils.gotoEVP();
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		ObjectAdmin.deleteAllCustomObjectsViaAPI();
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "EVP Portal");
+		}
+	}
+
+	@description = "LPS-162894 - Verify if the employee user can create a request after creating an organization"
+	@priority = 4
+	test CanCreateRequestAfterCreateOrganization {
+		task ("Given that the Employee user create an organization") {
+			EVPOrganization.goToCreateOrganization();
+
+			EVPOrganization.fillOrganizationForm(
+				bankingInfo = "Bank Test",
+				city = "City Test",
+				contactEmail = "test@liferay.com",
+				contactName = "Contact Test",
+				contactPhone = 12345,
+				country = "Country Test",
+				createRequest = "true",
+				organizationName = "Organization Test",
+				organizationSite = "www.test.com.br",
+				smallDescription = "Description Test",
+				state = "State Test",
+				street = "Street Test",
+				taxId = 12345,
+				zipCode = "12.345-67");
+		}
+
+		task ("When the employee user create a request") {
+			EVPRequest.fillRequestForm(
+				grantAmount = 20,
+				grantRequestType = "Proposed",
+				organization = "Organization Test",
+				phoneNumber = 12345,
+				requestBehalf = "Global/Local Poor",
+				requestDescription = "Request Description Test",
+				requestPurposes = "Basic Education",
+				requestType = "Grant");
+
+			EVPRequest.viewSuccessMessageRequest();
+		}
+
+		task ("Then the employee user view a Request Created and also view Organization Created") {
+			EVPNavigator.goToNavItem(navItem = "Request");
+
+			EVPUtils.viewStatus(
+				organizationName = "Organization Test",
+				status = "Awaiting Approval On EVP");
+
+			EVPNavigator.goToNavItem(navItem = "Organization");
+
+			EVPUtils.viewStatus(
+				organizationName = "Organization Test",
+				status = "Awaiting Approval On EVP");
+		}
+	}
+
+	@description = "LPS-162894 - Verify that after submit the form, when click on 'Done' button, will be redirected back to 'Organization' page"
+	@priority = 2
+	test CanSeeRedirectionToOrganizationPageAfterCreateOrganization {
+		task ("Given that the Employee user go to the organization page") {
+			EVPOrganization.goToCreateOrganization();
+		}
+
+		task ("When the employee user create an organization") {
+			EVPOrganization.fillOrganizationForm(
+				bankingInfo = "Bank Test",
+				city = "City Test",
+				contactEmail = "test@liferay.com",
+				contactName = "Contact Test",
+				contactPhone = 12345,
+				country = "Country Test",
+				done = "true",
+				organizationName = "Organization Test",
+				organizationSite = "www.test.com.br",
+				smallDescription = "Description Test",
+				state = "State Test",
+				street = "Street Test",
+				taxId = 12345,
+				zipCode = "12.345-67");
+		}
+
```

Comment:
You can remove this task and put this macro inside the first task: `"When the Employee user create a organization page"`